hazelcast-sql-core/src/test/java/com/hazelcast/sql/SqlOrderByTest.java

```diff
@@ -132,16 +110,16 @@ public void before() {
         Random r = ThreadLocalRandom.current();
         int nextNullValue = Math.max(1, r.nextInt(5));
         int nextSameValues = Math.max(1, r.nextInt(5));
+        long idx = Math.negateExact(DATA_SET_MAX_POSITIVE);
         int skipFirstEntries = 20;
```

Comment:
I'd rename it to `skipFirstPositiveEntries` because they're not first anymore.

Reply:
Thanks, fixed in [12c9391](https://github.com/hazelcast/hazelcast/pull/18113/commits/12c93911eb1d07f2742fe8a48110a0a56aa73ca5)